cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature

```diff
@@ -124,3 +124,77 @@ Feature: SecureStorage_Manage
             | Passing scope as null         | clear secure data values for an app with null scope    |
             | Passing scope as boolean      | clear secure data values for an app with boolean scope |
             | without scope                 | clear secure data values for an app without scope      |
+
+    @SecureStorage @manageSDK
+    Scenario Outline: SecureStorage.setForApp - Positive Scenario: <Scenario>
+        Given 1st party app invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for clearing stored value for an app'
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        Given 'secondary 3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
+        When 1st party app invokes the 'Firebolt' API to '<Set_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for '<Set_API_Key>'
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with '<Method_Validation_Key>'
+        Given 'secondary 3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with 'null for getting stored value'
+
+        Examples:
+            | Scenario                                            | Set_API_Key                                                                 | API_Key                                                                 | Method_Validation_Key                                                | Clear_API_Key                                       |
+            | setForApp with device scope with 2 apps             | set secure data value for an app with scope device                          | get stored value with scope as device and key as authTestTokenDevice    | expected value for authTestTokenDevice stored data in securestorage  | clear stored value with scope as device for an app  |
+
```

Comment:
we have already few TC with scope device right ? https://github.com/rdkcentral/firebolt-certification-suite/pull/144/files#diff-9820a1438090ff0eaf8f49c46ae1ec99c66731ceebcb27adb802170367498872R8 . can't we make use of that instead of adding new TC /

Reply:
Try if possible. Otherwise, we can have separate TC

Reply:
Thought of keeping the existing TCs as it is , because both are different scenarios like, one within the same app, and the other in different app.